app/Http/Controllers/TransportController.php

```diff
@@ -385,7 +385,7 @@ public static function TrainCheckin($tripId,
             'points'               => $trainCheckin->points,
             'alsoOnThisConnection' => $trainCheckin->alsoOnThisConnection,
             'lineName'             => $hafasTrip->linename,
-            'distance'             => $trainCheckin->distance,
+            'distance'             => $trainCheckin->distance / 1000,
```

Comment:
this should return meters and the frontend should catch it.

Reply:
That is currently too much effort for me. I don't want to search in a thousand places. (APIv0, APIv1, Frontend)

Anyway this method should just return the TrainCheckIn instead of an array.